airflow/providers/amazon/aws/auth_manager/aws_auth_manager.py

```diff
@@ -163,7 +179,7 @@ def is_authorized_view(
         access_view: AccessView,
         user: BaseUser | None = None,
     ) -> bool:
-        return self.avp_facade.is_authorized(
+        return self.is_logged_in() and self.avp_facade.is_authorized(
```

Comment:
instead of adding the call to `is_logged_in` before all the calls to `is_authorized` why not add the logged in check inside `is_authorized`?

Reply:
Good call :) Let me do that